build/package/Dockerfile

```diff
@@ -0,0 +1,29 @@
+# syntax=docker/dockerfile:1
+
```

Comment:
```
➜  k8s-deployment-scaler git:(development/go-server) docker images k8s-deployment-scaler
REPOSITORY              TAG       IMAGE ID       CREATED          SIZE
k8s-deployment-scaler   latest    959ac20a826b   21 minutes ago   306MB
```

This is incredibly large image for this kind of application. Would be good to trim it down a bit.

Reply:
I will update it to use `scratch` and include only the necessary binary and certificates in the final image.

```
k8s-deployment-scaler dereknorrbom$ docker images k8s-deployment-scaler
REPOSITORY              TAG       IMAGE ID       CREATED         SIZE
k8s-deployment-scaler   latest    007c9d25ea3b   4 seconds ago   7.72MB
```